.github/workflows/full-downstream.yml

```diff
@@ -0,0 +1,65 @@
+
```

Comment:
I guess this file and the pull-request ones need to be adjusted or ?
@Ginxo could you do so ? or any backport from `main` branch to be done ?